SEM_2_PRACTICALS/prac_06/guitars.py

```diff
@@ -0,0 +1,49 @@
+from SEM_2_PRACTICALS.prac_06.guitar import Guitar
+
+MIN_YEAR = 0
+MIN_COST = 0
+
+
+def main():
+    guitars = []
+    print("My guitars!")
+    get_guitars(guitars)
+    print("""
+    ... snip ...
+    """)
+    display_guitars(guitars,)
+
+
+def get_guitars(guitars):
+    name = input("Name: ").title()
+    while name != "":
+        year = int(get_valid_number("Year: ", MIN_YEAR))
+        cost = get_valid_number("Cost: $", MIN_COST)
+        guitar = Guitar(name, year, cost)
+        guitars.append(guitar)
+        print(f"{name} ({year}) : ${cost:,.2f} added.")
+        name = input("Name: ")
+    return guitars
+
+
+def get_valid_number(prompt, minimum):
+    number = float(input(prompt))
+    while number < minimum:
+        print("invalid number")
+        number = input(prompt)
+    return number
+
+
+def display_guitars(guitars):
+    print("These are my guitars")
+    for guitar in guitars:
+        if guitar.is_vintage():
+            vintage_status = "(vintage)"
+        else:
+            vintage_status = ""
+        index = guitars.index(guitar) + 1
```

Comment:
Why do you need this variable? You go backwards to figure out the index from the list, but you're in a for loop, and for loops iterate over a sequence in order, so you know the index already. See: 
https://github.com/CP1404/Starter/wiki/Programming-Patterns#repetition-structures